src/pages/Subscriptions/Subscriptions.tsx

```diff
@@ -0,0 +1,218 @@
+    min-width: 220px;
+    background: ${({ theme }) => theme.color.white};
+    border: 1px solid #c3cfd9;
+    margin: ${({ theme }) => theme.spacing.XXL};
+
+    h3 {
+        margin-bottom: ${({ theme }) => theme.spacing.S};
+    }
+
+    .infoContainer > h5:nth-of-type(1) {
+        height: 4rem;
+        margin-bottom: ${({ theme }) => theme.spacing.XL};
+    }
+    p {
+        margin-bottom: 0rem;
+    }
+    .infoContainer {
+        // border: 1px solid #c3cfd9; // DEBUG
+        text-align: center;
+        margin-bottom: ${({ theme }) => theme.spacing.M};
+    }
+    .listContainer {
+        // border: 1px solid #c3cfd9; // DEBUG
+    }
+    .hidden {
+        visibility: hidden;
+    }
+    .listContainerFree {
+        font-size: 1.1rem;
+        margin-top: -3rem;
+    }
+    .subscriptionButton {
+        text-decoration: none;
+        text-align: center;
+    }
+    ${SCREEN.MOBILE} {
+        flex-direction: column;
+        padding: ${({ theme }) =>
+            `${theme.spacing.XXL} 3rem ${theme.spacing.XXL} 3rem`};
+        width: 80%;
+        .hidden {
+            display: none;
+        }
+        .listContainerFree {
+            margin-top: 0rem;
+        }
+    }
+    ${SCREEN.DESKTOP} {
+        padding: ${({ theme }) =>
+            `${theme.spacing.XXXL} 3rem ${theme.spacing.XXXL} 3rem`};
+    }
+`
+
+export default function Subscriptions() {
+    const { t } = useTranslation('pricing') // NB: share the same .JSON than /Pricing
+    const history = useHistory()
+    // const dispatch = useDispatch()
+    // const showPricingModal = () => dispatch(showModal('PricingDetails')) //TODO: it works only with Newletters type -> see type.ts ti fix it
+
+    return (
+        <IonContent style={{ '--background': 'white' }}>
+            <MainView>
+                <h1>{t('ours-subscriptions')}</h1>
+                <Wrapper>
+                    <ColumnView>
+                        <div className="infoContainer">
+                            <h3>{t('free')}</h3>
+                            <h5>{t('meeting1')}</h5>
+                            <h5>{t('50-visios')}</h5>
+                            {/* trick: to have same heights in all columns: */}
+                            <div className="hidden">
+                                <h5>hidden</h5>
+                                <h5>hidden but with longer text</h5>
+                                <ButtonEstimate>hidden</ButtonEstimate>
+                            </div>
+                        </div>
+                        <div className="listContainer listContainerFree">
+                            <p>✔ {t('item1')}</p>
+                            <p>✔ {t('item2')}</p>
+                            <p>✔ {t('item3')}</p>
+                        </div>
+                        <ButtonSubscribe
+                            onClick={() => history.push('/pricing')}>
+                            {t('support-us')}
+                        </ButtonSubscribe>
```

Comment:
pls consider moving `ColumnView` different children into separate components, in order to make `Subscriptions` component lighter and easier to read

Reply:
also, you may consider using MaterialUi Grid to reduce the custom css